peachjam/views/article.py

```diff
@@ -10,7 +13,27 @@ class ArticleListView(ListView):
     template_name = "peachjam/article_list.html"
     context_object_name = "articles"
     navbar_link = "articles"
-    paginate_by = 5
+    paginate_by = 10
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        years = sorted(
+            list(
+                self.model.objects.filter(published=True)
+                .order_by()
```

Comment:
@nickmwangemi can we use the order_by here to sort straight from database without wrapping the list in `sorted` above?